src/js/components/FilterByService.js

```diff
@@ -93,8 +93,9 @@ var FilterByService = React.createClass({
         dropdownMenuListItemClassName="clickable"
         wrapperClassName="dropdown"
         items={this.getDropdownItems()}
-        onItemSelection={this.handleItemSelection}
         initialID={this.getSelectedId(this.props.byServiceFilter)}
+        onItemSelection={this.handleItemSelection}
+        ref={(ref) => this.dropdown = ref}
```

Comment:
docs say that method is on the way to being deprecated: https://facebook.github.io/react/docs/more-about-refs.html#the-ref-string-attribute ):